.github/workflows/ci.yml

```diff
@@ -73,21 +80,28 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.13.13]
-        java: [temurin@11]
+        scala: [2.13.14]
+        java: [temurin@17]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout current branch (full)
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Setup Java (temurin@11)
-        if: matrix.java == 'temurin@11'
+      - name: Setup Java (temurin@17)
+        if: matrix.java == 'temurin@17'
+        uses: actions/setup-java@v2
+        with:
+          distribution: temurin
+          java-version: 17
+
+      - name: Setup Java (temurin@21)
+        if: matrix.java == 'temurin@21'
         uses: actions/setup-java@v2
         with:
           distribution: temurin
-          java-version: 11
+          java-version: 21
```

Comment:
Java 21 is not in the publish matrix list, so `name: Setup Java (temurin@21)` section is redundant here

Reply:
This is automatically generated and  `sbtghactions.GenerativeKeys#githubWorkflowJavaVersions` does not offer a clean way of changing this. I've updated the plugin but it did not fix this. I don't think it makes sense to add an issue to the plugin repository when their intended way of choosing the Java version for publishing is different and we're publishing too rarely to bother fixing it.